packages/backend-for-frontend/src/services/purposeService.ts

```diff
@@ -276,9 +276,15 @@ export function purposeServiceBuilder(
       logger.info(
         `Creating purpose from ESErvice ${createSeed.eserviceId} and Risk Analysis ${createSeed.riskAnalysisId}`
       );
-      const payload = {
-        ...createSeed,
```

Comment:
Due to the spread, we were adding an extra `eserviceId` property together with the `eServiceId` expected one, and parsing on the body in purpose-process failed